src/plugins/data/public/query/query_string/language_service/query_result.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import './_recent_query.scss';
+
+import { BehaviorSubject } from 'rxjs';
+import { EuiButtonEmpty, EuiPopover, EuiText, EuiPopoverTitle } from '@elastic/eui';
+
+import React, { useState, useEffect } from 'react';
+
+export interface QueryStatus {
+  status: string;
+  message?: {
+    error?: string;
+    message?: string;
+    statusCode?: number;
+  };
+  time?: number;
```

Comment:
i think this should be `elapsedMs` or `timeElapsedMs` like how other parts of the application

Reply:
changed to elapsedMs